packages/front-end/pages/projects/index.tsx

```diff
@@ -54,15 +65,30 @@ const ProjectsPage: FC = () => {
         </div>
         <div style={{ flex: 1 }} />
         <div className="col-auto">
-          <button
-            className="btn btn-primary"
-            onClick={(e) => {
-              e.preventDefault();
-              setModalOpen({});
-            }}
+          <Tooltip
+            body={
+              <span className="premium">
+                <GBPremiumBadge /> Starter plans are limited to 1 project
```

Comment:
This also shows up for self-hosted open source users.  We typically only refer to "Starter" as the free cloud plan.  Maybe switch this language to say it requires a paid subscription.